tests/pagination/src/commonMain/graphql/schema.graphqls

```diff
@@ -1,5 +1,7 @@
 type Query {
-  users(first: Int = 10, after: String = null, last: Int = null, before: String = null): UserConnection!
+  usersCursorBased(first: Int = 10, after: String = null, last: Int = null, before: String = null): UserConnection!
+  usersOffsetBased1(offset: Int = 0, limit: Int = 10): [User!]!
```

Comment:
Alas, I can't find a way to make this example work with the recent API changes. The fact that it returns an a list directly and not an object containing the list makes things difficult.